src/relayclient/RelayClient.ts

```diff
@@ -15,9 +15,14 @@ import RelayedTransactionValidator from './RelayedTransactionValidator'
 import { configureGSN, getDependencies, GSNConfig, GSNDependencies } from './GSNConfigurator'
 import { RelayInfo } from './types/RelayInfo'
 
-// generate "approvalData" for a request. must return string-encoded bytes array
-export const EmptyApprovalData: AsyncApprovalData = async (): Promise<PrefixedHexString> => {
-  return await Promise.resolve('0x')
+export const ZERO_BYTES32 = '0x'.padEnd(66, '0')
```

Comment:
Aaand you won't be needing this of course

Reply:
might want to move it to some constants file, but we don't have any in client, only in test.

Reply:
remove

Reply:
removed